packit/constants.py

```diff
@@ -190,6 +190,11 @@
     "```\ngit fetch https://src.fedoraproject.org/forks/{user}/rpms/{package}.git "
     "refs/heads/*:refs/remotes/{user}/*\n"
     "git checkout {user}/{branch}\n```\n"
+    "\n\n"
+    "Before pushing builds/updates, please remember to:\n"
```

Comment:
```suggestion
    "Before pushing builds/updates, please remember to (mainly):\n"
```

I would be more explicit that those are not the **only** responsibilities of the maintainers.